src/core/public/application/application_service.tsx

```diff
@@ -117,6 +118,11 @@ export class ApplicationService {
       return replace ? this.history!.replace(url, state) : this.history!.push(url, state);
     };
 
+    this.navigateToNewTab = (url) => {
+      // window.open copies session information if base url is same
+      return window.open(basename + url, '_blank');
```

Comment:
NIT: use `appendAppPath(basename, url)`

Reply:
Updated in [`c1d3f3f` (#96723)](https://github.com/elastic/kibana/pull/96723/commits/c1d3f3f0cf8e8258c02a393daa71c74ca161fc46)